packages/@stylexjs/postcss-plugin/src/index.js

```diff
@@ -22,6 +22,7 @@ const plugin = ({
   include,
   exclude,
   useCSSLayers = false,
+  importSources = ['@stylexjs/stylex', 'stylex'],
```

Comment:
Strictly speaking, we only need `"stylex"` to maintain backward compatibility as that was the default. In other places like eslint plugin, we use `@stylexjs/stylex` as the default value so I added both `@stylexjs/stylex` and `stylex` to reduce confusion.

If we're okay with breaking some backward compatibility, we can just use `@stylexjs/stylex` to align.